samplekotlin/build.gradle

```diff
@@ -44,4 +46,5 @@ dependencies {
     implementation Dependencies.coil
     implementation Dependencies.appCompat
     implementation Dependencies.constraintLayout
+    implementation 'androidx.activity:activity-ktx:1.1.0'
```

Comment:
Abstract this to `Dependencies` if we keep

Reply:
@JcMinarro why do we use this build system? seems annoying to sync to the tutorial where we use the old school approach

@carterhudson the biggest question here is how many people are used to the syntax activity-ktx introduces. lets ask the team. i think its pretty common to use this, but not sure...

Reply:
it's much easier to manage dependencies when we have more than one module in the repository

Reply:
It might be a good idea to inline the dependency coordinates and versions here in the tutorial instead of having them in `Dependencies`. It adds unnecessary extra complexity to the tutorial / sample code if someone's not familiar with the `buildSrc` approach yet.

It's more work for us, but it makes things friendlier and simpler for the users taking a look at the project, which is absolutely something we should strive for. Easier to copy-paste them if they're inlined, too.

Reply:
One thing that I do like about inlined dependencies is that you get linting around dependencies that are outdated (highlighted) and using option + enter on `+` to have it resolve the latest instead of having to go look at versioning.